holmes/plugins/toolsets/grafana/base_grafana_toolset.py

```
@@ -4,7 +4,10 @@
 from holmes.core.tools import CallablePrerequisite, Tool, Toolset, ToolsetTag
 from holmes.plugins.toolsets.consts import TOOLSET_CONFIG_MISSING_ERROR
 from holmes.plugins.toolsets.grafana.common import GrafanaConfig
-from holmes.plugins.toolsets.grafana.grafana_api import get_health
+
+from holmes.plugins.toolsets.grafana.grafana_api import grafana_health_check
+from holmes.plugins.toolsets.utils import TOOLSET_CONFIG_MISSING_ERROR
```

Ruff (0.11.9)

Redefinition of unused `TOOLSET_CONFIG_MISSING_ERROR` from line 5

(F811)